### PR TITLE
Update Readme for EdgeIdentity

### PR DESCRIPTION
**Pull request by cacheung**
Add more APIs
Include the link to gitbook EdgeIdentity FAQ 
Add info/link for when have core identity and EdgeIdentity in the same app.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/edgeidentity/README.md [hunk 1]
```diff
@@ -94,18 +97,201 @@ public class MainApplication extends Application implements ReactApplication {
...
+
+### AuthenticatedState
+Defines the state an Identity Item is authenticated for.
+
+The possible authenticated states are:
+* Ambiguous - the state is ambiguous or not defined
+* Authenticated - the user is identified by a login or similar action
+* LoggedOut - the user was identified by a login action at a previous time, but is not logged in now
+
+//TO DO: Add Syntax
```
Comment (emdobrin) [inline packages/edgeidentity/README.md:283]: let's update this

File: packages/edgeidentity/README.md [hunk 2]
```diff
@@ -94,18 +97,201 @@ public class MainApplication extends Application implements ReactApplication {
...
+
+
+## Frequently Asked Questions (FAQ)
```
Comment (emdobrin) [inline packages/edgeidentity/README.md:286]: I think we should remove this section and only point to the GitBook FAQ page, what do you think?

File: packages/edgeidentity/README.md [hunk 3]
```diff
@@ -94,18 +97,201 @@ public class MainApplication extends Application implements ReactApplication {
...
 
+```javascript
+static extensionVersion(): Promise<string>
+```
+**Example**
 ```javascript
 AEPIdentity.extensionVersion().then(version => console.log("AdobeExperienceSDK: AEPEdgeIdentity version: " + version));
 ```
-#### Get Experience Cloud ID:
+### Get Experience Cloud ID:
```
Comment (emdobrin) [inline packages/edgeidentity/README.md:119]: For these API reference headers can we use the API name and order them alphabetically? e.g. getExperienceCloudId , getIdentities ..

File: packages/edgeidentity/README.md (hunk 2 above)
Comment (cacheung) [reply packages/edgeidentity/README.md]: I left the FAQ section with  referring to the gitbook link only.  Sounds good?

**Comment by emdobrin** [comment]
@cacheung nice work, the docs look much cleaner now.

File: packages/edgeidentity/README.md [hunk 4]
```diff
@@ -24,88 +24,215 @@ npm install @adobe/react-native-aepedgeidentity
...
+let identityItem  = new AEPIdentityItem("user@example.com");
+AEPIdentity.removeIdentity(identityItem, "Email");
+```
+
+### resetIdentity:
+Clears all identities stored in the Identity extension and generates a new Experience Cloud ID (ECID) . Using this API does not remove the identifiers from the server-side User Profile Graph or Identity Graph.
+
+:information_source: The Identity for Edge Network extension does not read the Mobile SDK's privacy status and therefor setting the SDK's privacy status to opt-out will not clear the identities from the Identity for Edge Network extension.
+
+See [reset identities api](https://github.com/emdobrin/aepsdk-react-native/tree/consent/packages/core#reset-identities) for more details.
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply packages/edgeidentity/README.md]: we should main repo and branch https://github.com/adobe/aepsdk-react-native/tree/main/packages/core#reset-identities

**Review by emdobrin — APPROVED** [review]
Looks good with couple small unresolved comments.

**Pull request merged**